app/models/viewer.rb

```diff
@@ -0,0 +1,10 @@
+##
+# Model for viewer
+class Viewer < ActiveRecord::Base
+  belongs_to :exhibit, class_name: Spotlight::Exhibit
+
+  def to_partial_path
+    return 'oembed_default' if viewer_type == 'sul-embed'
+    "../viewers/#{viewer_type}"
```

Comment:
Nope. It looked for the partials at `spotlight/...`, which is at the same level as `viewers`